src/transforms/vue/variants.ts

```diff
@@ -74,12 +74,12 @@ export function resolveVariantsProps(variants, isTs: boolean) {
 
       const isBooleanVariant = Object.keys(variant).some(key => (key === 'true' || key === 'false'))
       if (isBooleanVariant) {
-        prop.type = isTs ? ' [Boolean, Object] as PropType<boolean | { [key in PinceauMediaQueries]?: boolean }>' : ' [Boolean, Object]'
+        prop.type = isTs ? ' [Boolean, Object] as PropType<boolean | { [key in import(\'pinceau\').PinceauMediaQueries]?: boolean }>' : ' [Boolean, Object]'
```

Comment:
Note:
- `PropType` -> `import('vue').PropType`
- Add `as const` for `type` and `default.`